packages/base/src/StaticArea.js

```diff
@@ -0,0 +1,97 @@
+let staticAreaIndex = 1;
+class StaticAreaItem extends HTMLElement {
```

Comment:
Split this into a new file: StaticAreaItem.js:
- dynamic class
- one instance per web component that needs it
- it calls window.customElementsRegistry.define directly before the export